sql/mob_skills.sql

```diff
@@ -582,11 +582,11 @@ INSERT INTO `mob_skills` VALUES (755,909,'fission',0,7.0,2000,1500,4,0,0,0,0,0,0
 -- INSERT INTO `mob_skills` VALUES (759,503,'howl',0,7.0,2000,1500,4,0,0,0,0,0,0);
 -- INSERT INTO `mob_skills` VALUES (760,504,'howl',0,7.0,2000,1500,4,0,0,0,0,0,0);
 -- INSERT INTO `mob_skills` VALUES (761,505,'howl',0,7.0,2000,1500,4,0,0,0,0,0,0);
--- INSERT INTO `mob_skills` VALUES (762,354,'howl',0,7.0,2000,1500,4,0,0,0,0,0,0);
+INSERT INTO `mob_skills` VALUES (762,354,'howl',0,7.0,2000,1500,4,0,0,0,0,0,0); -- Quadav
```

Comment:
Is `mob_prepare_time` of 4 intended?

Reply:
omg welcome back @ibm2431  <3

Reply:
@ibm2431 that looks like `mob_valid_targets` is 4 there, prepare time the 1500 next to it. both look to be preexisting copypaste values (which are unlikely to be correct for most skills)

in this case
```cpp
 TARGET_ENEMY = 0x04,
```
so that Quadav will buff me instead of itself! COME AT ME TURTLE MAN!

Reply:
> @ibm2431 that looks like `mob_valid_targets` is 4 there, prepare time the 1500 next to it. both look to be preexisting copypaste values (which are unlikely to be correct for most skills)
> 
> [in this case](https://github.com/project-topaz/topaz/blob/8c722838094c1aac0887eb3f13a3eb3168926933/src/map/entities/battleentity.h#L358)
> 
> ```c++
>  TARGET_ENEMY = 0x04,
> ```
> 
> so that Quadav will buff me instead of itself! COME AT ME TURTLE MAN!

lol i love your comments @TeoTwawki  they always cheer up my day =)

Reply:
@kaincenteno I have invented a new sport. I call it turtle tipping. to play, turn off mob despawning. go deep into palborough mines, build up tp. anger a turtle on the other side of the gates where Zi'gi Boneeater spawns. he'll have to run all the way around to get to you. while you wait, anger a turtle on your side till he howls at you. All his friends will arrive just in time to eat a big pile of howl boosted aoe ws to the face.